src/atom_finder/classifier/decrement-increment.clj

```diff
@@ -0,0 +1,33 @@
+(in-ns 'atom-finder.classifier)
+(import '(org.eclipse.cdt.core.dom.ast IASTNode IASTUnaryExpression))
+
+
+(defn decrement-increment?
+  "Is this oprator decrement or increment?"
+  [node]
+  (let [u-ops #{ ; all decrement or increment operators
+                IASTUnaryExpression/op_postFixDecr      IASTUnaryExpression/op_postFixIncr
+                IASTUnaryExpression/op_prefixDecr       IASTUnaryExpression/op_prefixIncr}]
```

Comment:
Prefix increment and decrement are actually a separate atom. If you want, it'd be great to split this out into two classifiers, but as is, this is actually misclassifying ++x as "post-increment"